### PR TITLE
Php version testing

### PR DESCRIPTION
**Pull request by blong14**

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# PHP OpenTracing API - PHP Version Testing
+
+## Current PHP Support
+- 5.6
+- 7.0
+- 7.1
+- 7.2
+
```
Comment (jonahgeorge) [inline TESTING.md]: | version | status |
|---------|--------|
| 5.6     | 𝙓      |
| 7.0     | ✔      |
| 7.1     | ✔      |
| 7.2     | ✔      |

```
| version | status |
|---------|--------|
| 5.6     | 𝙓      |
| 7.0     | ✔      |
| 7.1     | ✔      |
| 7.2     | ✔      |
```

**Comment by blong14** [comment]
@jonahgeorge 7.0 runs as expected now. The only tests that fails is 5.6 and the failure I think is expected as we haven't updated the code yet.

**Comment by blong14** [comment]
@jonahgeorge linting now fails the build. Not sure if thats exactly what we want or not. I can update code in this PR to handle the lint warnings or add that to the eventual 5.6 update. Thoughts?

**Comment by jonahgeorge** [comment]
Sweet. I'm happy merging this in and then cutting the next release when the lint failures have been cleaned up. Lastly, would you mind putting an `allowed_failure` block on 5.6?

https://docs.travis-ci.com/user/customizing-the-build/#Rows-that-are-Allowed-to-Fail

**Pull request closed without merging**